Assets/Scripts/MainMenu/SoundSettingsManager.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+//Controls for sound settings
+public class SoundSettingsManager : MonoBehaviour {
+	public Slider masterVolumeSlider;
+	public Slider musicVolumeSlider;
+	public Slider soundEffectsVolumeSlider;
+	
+	public SoundSettings soundSettings;
+
+	void OnEnable()
+	{
+		soundSettings = new SoundSettings();
+
+		masterVolumeSlider.value = AudioListener.volume;
+
+		masterVolumeSlider.onValueChanged.AddListener(delegate {OnMasterVolumeChange();});
+		musicVolumeSlider.onValueChanged.AddListener(delegate {OnMusicVolumeChange();});
+		soundEffectsVolumeSlider.onValueChanged.AddListener(delegate {OnSoundEffectVolumeChange();});
+	}
+
+	public void OnMasterVolumeChange(){
+		soundSettings.masterVolume = AudioListener.volume = masterVolumeSlider.value;
+	}
+
+	public void OnMusicVolumeChange()
+	{
+		//TODO

```

Comment:
Describe tasks of todo, or if they are unused (default template) remove/comment why